tests/cases/fourslash/docCommentTemplateWithMultipleJSDocAndParameters.ts

```diff
@@ -9,4 +9,8 @@
 /////*/**/
 ////function foo(p) {}
 
-verify.docCommentTemplateAt("", 3, "/** */");

```

Comment:
I think, in this case, we need to provide a template because the JSDoc comment that contains tags is not related to the host it is attached to. Or am I wrong? @DanielRosenwasser What do you think?

Reply:
That sounds right to me because of the intervening `/** */`.

However, you should also test the case without an intervening empty comment:

```
/** @param p */
/*/**/
function foo(p) { }
```

The seems like code that should not generate the template, but the current code might. I'm not sure.
(It's also possible that starting a fresh comment below the existing one implies that the author *always* wants a fresh template, but I think it's against the previous intent of the code in jsDoc.ts.)